app/models/order.rb

```diff
@@ -0,0 +1,7 @@
+class Order < ApplicationRecord
+  belongs_to :customer
+
+  validates :product_name, presence: true
+  validates :product_count, presence: true
```

Comment:
it looks like we didn't say you had to validate that product_count was a number on the model side (only the database/schema side) so all is good! but just so you know, if you try to enter a string or something it will still save, but store the value as 0.

Reply:
consider using `numericality: { only_integer: true }` to ensure `product_count` is an integer.